template/{{cookiecutter.project_name}}/taskcluster/ci/hello/kind.yml

```diff
@@ -1,10 +1,6 @@
 ---
-loader: taskgraph.loader.transform:loader
-
 transforms:
   - {{cookiecutter.project_slug}}_taskgraph.transforms.hello:transforms
-  - taskgraph.transforms.job:transforms
-  - taskgraph.transforms.task:transforms
```

Comment:
The only downside to this is that things aren't as transparent anymore. It won't be obvious that the `job` then the `task` transforms are applied in that order.

I think longer term we'll want to combine these into a single `TransformSequence`, but let's leave that for a follow-up.